shell/platform/windows/flutter_windows_engine_unittests.cc

```diff
@@ -1169,5 +1170,30 @@ TEST_F(FlutterWindowsEngineTest, ChannelListenedTo) {
   }
 }
 
+TEST_F(FlutterWindowsEngineTest, ReceivePlatformViewMessage) {
+  FlutterWindowsEngineBuilder builder{GetContext()};
+  builder.SetDartEntrypoint("sendCreatePlatformViewMethod");
+  auto engine = builder.Build();
+
+  EngineModifier modifier(engine.get());
```

Comment:
```suggestion
  EngineModifier modifier{engine.get()};
```

Reply:
Do we have a style guide to prefer this syntax?

Reply:
I couldn't find one, but @cbracken has suggested this before on my PRs. My (admittedly weak) understanding is this helps disambiguate between method calls and initializations.